src/RoundTabler/PCIScan.java

```diff
@@ -40,24 +43,25 @@ public PCIScan(Connection MYSqlDBConnection, String TableName  ){
 		if ( pTableName.compareTo("*") == 0){
 			// We are scanning all tables
 			// so select should retrieve all table all columns which can store text
-			pMYSQLColumnSelect = "SELECT TABLE_NAME, COLUMN_NAME from information_schema.COLUMNS where table_schema=DATABASE() and DATA_TYPE in " + this.pMySQLColumnTypes + ';';
+			pMYSQLColumnSelect = "SELECT TABLE_NAME, COLUMN_NAME from information_schema.COLUMNS where table_schema=DATABASE() and DATA_TYPE in " + this.pMYSQLColumnTypes + ';';
 
 
 		} else {
 			// We are scanning a specific table
 			// so select should retrieve all table all columns which can store text
-		    pMYSQLColumnSelect = "SELECT TABLE_NAME, COLUMN_NAME from information_schema.COLUMNS where table_schema=DATABASE() and TABLE_NAME = '" + pTableName + " and DATA_TYPE in "  + this.pMySQLColumnTypes + ';';
+		    pMYSQLColumnSelect = "SELECT TABLE_NAME, COLUMN_NAME from information_schema.COLUMNS where table_schema=DATABASE() and TABLE_NAME = '" + pTableName + " and DATA_TYPE in "  + this.pMYSQLColumnTypes+ ';';
 
 
 		}
 
 	}
 
-	
-	public void ScanMYSQLTables(){
+	public String ScanResult() { return this.psbResults.toString() };
```

Comment:
add semicolon after toString() inside the function - remove the semicolon at the end

Reply:
at line 38, ScanResult is not declared anywhere so it errors out - what is the variable type supposed to be here?